website/index.html

```diff
@@ -107,6 +107,33 @@ <h3 id="examples">Examples</h3>
 }
 </pre>
 
+            <h4>Try/Finally example</h4>
```

Comment:
I think this might be a distraction from our otherwise spartan docs. Wanna just fix the first example to use try/finally?